src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4537,7 +4537,7 @@ void os::Linux::disable_numa(const char* reason) {
   if ((UseNUMA && FLAG_IS_CMDLINE(UseNUMA)) ||
       (UseNUMAInterleaving && FLAG_IS_CMDLINE(UseNUMAInterleaving))) {
     // Only issue a warning if the user explicitly asked for NUMA support
-    log_warning(os)("NUMA support disabled: %s", reason);
+    log_info(os)("NUMA support disabled: %s", reason);
```

Comment:
So this is too coarse. Some reasons for disabling NUMA may require a warning because something is actually wrong. I only want to see the new warning for "Only a single NUMA node is available" not be a warning.